File: C/pythagoras.c
#include <stdio.h>//working....
#include <math.h>//success.......
int main(){
	int side2,side1;
	printf("we will use pythagoras theorem\nand find the 3rd side");
	printf("enter 1st side and 2nd side\n");
	scanf("%d\n%d",&side1,&side2);
	double newside1=pow(side1,2);
    double newside2=pow(side2,2);
    double final=sqrt(newside1+newside2);
printf("the third side is %lf\n",final );
}

File: java/pythagoras.java
import java.util.*;//success....
public class pythagoras{//working......
	public static void main(String sap[]){
		double side1,side2,user;
		System.out.println("we will use pythagoras theorem\nand will find the third side");
		System.out.println("enter the 1st side and the 2nd side");
		Scanner nrt=new Scanner(System.in);
        side1=nrt.nextDouble();
        side2=nrt.nextDouble();
        user=Math.sqrt((Math.pow(side1,2))+(Math.pow(side2,2)));
        System.out.println("the third side is.."+user);
        nrt.close();
	}
}